.changeset/six-humans-lick.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fake-scope/fake-pkg": patch
```

Comment:
this is a bug in the changeset bot :( 

see https://github.com/changesets/bot/issues/82

```suggestion
"@livekit/protocol": patch
```